### PR TITLE
FFM-1758: adds ff dashboard topic

### PR DESCRIPTION
**Pull request by StrawberryPad**
[**Link to preview**](https://6453955584e2bc11339ce2c2--harness-developer.netlify.app/docs/feature-flags/ff-data/dashboard)

### CODE REVIEW

**Review by brian-f-harness — DISMISSED** [review]
lgtm. Added a couple suggestions.

File: docs/feature-flags/ff-data/dashboard.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+---
+title: Use the FF dashboard
```
[the remark this answers is not in the document]
Comment (StrawberryPad) [reply docs/feature-flags/ff-data/dashboard.md:2]: I checked it's OK to use this with Ethan. They use it in prod/marketing as well. :-)

File: docs/feature-flags/ff-data/dashboard.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+## Feature Flags dashboard highlights
+
+### View the dashboard
+
+To view the Feature Flags dashboard: 
+
+1. In Harness, select **Dashboards** in the left navigation pane.
+1. In the Dashboards page, select **Feature Flags**, and then choose **Feature Flag Analytics**.
+
+	The dashboard is displayed, showing the total number of flags, targets, and environments in this Harness organization.
```
[the remark this answers is not in the document]
Comment (StrawberryPad) [reply docs/feature-flags/ff-data/dashboard.md:23]: I actually hate this wrongful use of 'displays' as an intransitive verb. It either displays _something_, or is displayed. :-)

File: docs/feature-flags/ff-data/dashboard.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+### View flags created by users
+
+This chart is helpful to see who in your organization is creating flags, and how many. One way to use this is to help adoption of feature flag usage by having the people who create a lot of flags help others to use them.
+
+![](./static/dashboard-flags-by-user.png)
+
+### View flags and targets created over time
+
+You should see an increase in flags and targets created over time. The targets created gives an idea of how many users are potentially affected by your flags. As your organization's use of flags increases, the number of targets should increase. These charts help you visualize this.
```
[the remark this answers is not in the document]
Comment (StrawberryPad) [reply docs/feature-flags/ff-data/dashboard.md:64]: "See" is used metaphorically here.

**Pull request merged**